app/javascript/bundles/Identity/components/KnownIdentity.jsx

```diff
@@ -26,7 +27,7 @@ export default class KnownIdentity extends React.Component {
         <div className="left">
           <Table>
             <TableRow headerLabel="ID No.">
-              <b>{id}</b>
+              <b>{identity_number}</b>
```

Comment:
what's in {identity_number}?

Reply:
it is the second item in the identities array list in that picture of the data you showed, is that not what you meant?

Reply:
We're removing identity number
Replacing with passport number


![image](https://user-images.githubusercontent.com/12266/49843923-c2307200-fe25-11e8-8243-5861351a64c8.png)

Reply:
there is identitiy_number and serial_number

Reply:
have you kept the old variable name.. ? I'm way confused why identity_number is in there still.